roles/thanos_sidecar/templates/docker-compose.yml.j2

```diff
@@ -0,0 +1,27 @@
+---
+services:
+  thanos_sidecar:
+    image: "{{ thanos_sidecar_image }}"
```

Comment:
This parameter is missing in defaults.yml

Reply:
It is present; observe: https://github.com/osism/ansible-collection-services/pull/1221/files#diff-d06bd04b4a690f9e61ad17a276f8869f6d3fb3a5e2cde74460521877a44c9a8aR34